EDMarketConnector.py

```diff
@@ -1489,6 +1565,11 @@ def crewroletext(role: str) -> str:
                 if not should_return:
                     self.w.after(int(SERVER_RETRY * 1000), self.capi_request_data)
 
+            if entry['event'] in ['CarrierBuy', 'CarrierStats', 'CarrierTradeOrder']:  # 'CargoTransfer' too?
```

Comment:
Re: `CargoTransfer` ... I'm conflicted on this because whilst it's a pro to trigger the query after cargo hold changes ... doing so on the first of possibly many would 'jump the gun' and only updated for the first... and that's assuming the CAPI data updates in a timely manner.

The latter also applies to `CarrierTradeOrder`.  I'm leaning towards requiring the player to relogin in the game (to get `CarrierStats`) as the simplest way to give clear player control for this, without complicating things with an extra UI button/config options.

Reply:
Ok, will remove that comment and also remove `CarrierTradeOrder` as a trigger.